### ISSUE TITLE
NewType requires a second argument

### ISSUE THREAD

**Issue by not-my-profile** [microsoft/pyright#4374]
Pyright doesn't complain about the following code:

```python
T = NewType('T')
```

despite it leading to a runtime TypeError:

> TypeError: NewType() missing 1 required positional argument: 'tp'

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.286, which I just published. It will also be included in a future version of pylance.
